### DISCUSSION TITLE
I want to get the scanned qr code position (corner offsets), but I have a problem.

### DISCUSSION THREAD

**Discussion by Felypee** [juliansteenbakker/mobile_scanner#498] [Q&A]
When I scan a qr code and the onDetect function sends me the scanned barcode with the list of offsets  in the barcode corners property, I notice that the x value and y value is a different number than what I have in the screen.

For example, When scanning I put the qr code top left corner in the screen top left corner, it shoud have an offset of (0,0) but I don't understand why is sending other numbers like (57.0, 63.0).

I notice that when I move the phone away from the screen the numbers change.

Let me know if something is missing in my analysis. Thanks!

**Comment by raymond-holmboe** [comment]
Check out this working example in the repo: example/lib/barcode_scanner_window.dart
